api/data_refinery_api/views.py

```diff
@@ -250,6 +261,18 @@ def perform_update(self, serializer):
             # We could be more aggressive with requirements checking here, but
             # there could be use cases where you don't want to supply an email.
             supplied_email_address = self.request.data.get('email_address', None)
+            if supplied_email_address and MAILCHIMP_API_KEY and RUNNING_IN_CLOUD:
+                try:
+                    client = mailchimp3.MailChimp(mc_user=MAILCHIMP_USER, mc_api=MAILCHIMP_API_KEY)
+                    data = {
+                        "email_address": supplied_email_address,
+                        "status": "subscribed"
+                    }
+                    client.lists.members.create(MAILCHIMP_LIST_ID, data)
+                except mailchimp3.mailchimpclient.MailChimpError as mc_e:
+                    pass # This is likely an user-already-on-list error. It's okay.
+                except Exception as e:
+                    pass # Something outside of our control has gone wrong. It's okay.
```

Comment:
Could we get a log line here so that if this is happening every time we'll know?